config/kubernetes/staging/config_map.yml

```diff
@@ -13,3 +13,5 @@ data:
   OMNIAUTH_AZURE_CLIENT_ID: 5d1ba28b-234a-44e1-9464-f5bec5228bfc
   OMNIAUTH_AZURE_TENANT_ID: c6874728-71e6-41fe-a9e1-2e8c36776ad8
   OMNIAUTH_AZURE_REDIRECT_URI: https://staging.review-criminal-legal-aid.service.justice.gov.uk/users/auth/azure_ad/callback
+  PHASE_BANNER_TAG: staging
```

Comment:
Can we not use the already existing `ENV_NAME` to imply the name and the colour?
I feel these 2 additional ENV vars are unnecessary.

Reply:
Discussed and agreed to remove custom env variables and use existing `HostEnv` variable